spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -2289,8 +2290,7 @@ definitions:
                 - bits 0-4: Signal health status. See IS-GPS-200H
                   Table 20-VIII. Codes for Health of SV Signal
                   Components.
-              Satellite health status for GLO:
-                See GLO ICD 5.1 table 5.1 for details
+              Satellite health status for GLO, See GLO ICD 5.1 table 5.1 for details:
                 - bit 0: C(n), "unhealthy" flag that is transmitted within
```

Comment:
Just noting that descriptions like this one, with lists embedded, appear in the pre-existing `sbp.pdf` as a single paragraph of unformatted text.  Likewise with the comments in Haskell.  That is obviously not what is intended, however there are not many in the yaml, and the resulting rendering is still usable, so this PR is not going to try to fix that.

In C/Python/Java at least, the EOL are preserved, but additional lines are not aligned with indent of the first line.

Changing these from `|` string blocks to `>` string blocks does not cause them to become a single paragraph in the C/Java/etc - the indented lines are still treated as pre-formatted.

This patch will fix the alignment of them in C/Python/Java, however it cant autowrap the content inside the bullet points - they still need to be pre-formatted in the YAML.

Reply:
👍